test/spec/tuples.wast

```diff
@@ -0,0 +1,9 @@
+(assert_invalid
+  (module
+    (func $foo
+      (local $temp ((ref func) i32))
+    )
+  )
+  "var must be defaultable"
```

Comment:
This would also be a good test to put in `test/lit/validation` if you want. (But LGTM either way.)

Reply:
Interesting, thanks, I'll look at that later.